promql/testdata/legacy.test

```diff
@@ -308,6 +308,30 @@ eval instant at 50m round(0.045 * http_requests{group="production",job="api-serv
 	{group="production", instance="0", job="api-server"} 5 
 	{group="production", instance="1", job="api-server"} 10 
 
+eval instant at 50m clamp_max(http_requests{group="production",job="api-server"}, +Inf)
```

Comment:
Could you add these tests to https://github.com/prometheus/prometheus/blob/master/promql/testdata/functions.test instead? `legacy.test` was for old tests carried over from a previous way of testing expressions.